packages/file_selector/file_selector/test/file_selector_test.dart

```diff
@@ -13,10 +13,12 @@ void main() {
   const String confirmButtonText = 'Use this profile picture';
   const String suggestedName = 'suggested_name';
   final List<XTypeGroup> acceptedTypeGroups = <XTypeGroup>[
+    // ignore: prefer_const_constructors
```

Comment:
These should have a TODO comment linking to the relevant issue so that it's easy to find them all to remove them again (and to leave context for someone to understand why it was there if we miss any).